kartothek/io/testing/read.py

```diff
@@ -31,6 +31,7 @@
 """
 
 import datetime
+import functools
```

Comment:
partial is already importet 2 lines below. That appers to be the only usage of this new import

Reply:
OK. Changed the test to use `partial` instead of `functools.partial` and removed the import.